WooCommerce/Classes/ViewRelated/Orders/Cells/OrderNoteTableViewCell.swift

```diff
@@ -80,25 +76,29 @@ private extension OrderNoteTableViewCell {
     /// Updates the Status Label + Icon's Color
     ///
     func noteTypeWasUpdated() {
+        let theAuthor = author ?? ""
+        let dateOfCreation = dateCreated ?? ""
         if isCustomerNote {
             iconButton.backgroundColor = StyleManager.statusPrimaryBoldColor
-            statusLabel.text = NSLocalizedString("Note to customer", comment: "Labels an order note to let user know it's visible to the customer")
+            let localizationComment = "Labels an order note to let user know it's visible to the customer. Reads like 05:30 PM - username (To Customer)"
+            statusLabel.text = NSLocalizedString("\(dateOfCreation) - \(theAuthor) (To Customer)", comment: localizationComment)
```

Comment:
To follow best practices for localization, please use `String.localizedStringWithFormat()`. String interpolations can cause a lot of errors, which can lead to crashes.

Example:
**OK:**
``` 
let year = 2019
let template = NSLocalizedString("© %d Automattic, Inc.", comment: "Copyright Notice")
let str = String.localizeStringWithFormat(template, year)
```

**Not OK:**
```
let year = 2019
let str = NSLocalizedString("© \(year) Automattic, Inc.", comment: "Copyright Notice")
```